modules/tooling-support-parent/tooling-support/src/main/java/org/mule/runtime/module/tooling/api/artifact/DeclarationSession.java

```diff
@@ -70,6 +72,19 @@
    */
   MetadataResult<ComponentMetadataTypesDescriptor> resolveComponentMetadata(ComponentElementDeclaration component);
 
+  /**
+   * Retrieves any sample data available for the component.
+   *
+   * @param component the component whose sample data is required
+   * @return a {@link SampleDataResult} with the sample data message
+   */
+  SampleDataResult getSampleData(ComponentElementDeclaration component);
+
+  /**
+   * Exposes the application Expression Manager.
+   */
+  ExpressionManager getExpressionManager();
```

Comment:
Why do we need this? How will the client of the DeclarationSession need the ExpressionManager?

Reply:
We have defined to do it this way just to validate the approach for implementing this feature. I would say that we should create a task to enable access to the org.mule.runtime.api.el.DefaultExpressionLanguageFactoryService from our Agent plugin so we can create a org.mule.runtime.api.el.ExpressionLanguage which part of Mule API instead of exposing the one associated to the session. 
Let's create a Jira to do this on next sprint.

Reply:
[CMTS-70: Provide access to Expression Language from Tooling Agent](https://www.mulesoft.org/jira/browse/CMTS-64)